### ISSUE TITLE
Clarify digest-algorithm syntax

### ISSUE THREAD

**Issue by ioggstream** [ioggstream/draft-polli-resource-digests-http#34]
## I expect

- the `BNF` to be updated to the current standard (ABNF?)

https://github.com/ioggstream/draft-polli-resource-digests-http/blame/master/draft-polli-resource-digests-http.md#L302

- some examples of passing parameters to the algorithms

**Comment by ioggstream** [comment]
Taken from https://github.com/martinthomson/http-mice/blame/537cc8b7f61787306d358f14178b146eae631ee4/draft-thomson-http-mice.md#L245

```
The MI header field uses the extended ABNF syntax defined in Section 1.2 of
[RFC7230] and the `parameter` rule from [RFC7231]:
```

we could say that the Digest header takes parameter, instead of the digest-algorithm: this would be similar to `Want-Digest`

```
Digest: new-algo=f1b2c3;rs=5, sha-256=fbfbfba
```

**Comment by LPardue** [comment]
If I understand correctly, this seems like two indpendent issues:

1) Ensure that the syntax definition aligns with modern best practice. Whether that be RFC723x, HTTP core, or Structured headers.

2) Consider adding parameters to the Digest header. Do we have any examples in the wild that do this, or use cases that want to do this?

**Comment by ioggstream** [comment]
I try to clarify. RFC3230 stated that:

```
   For some algorithms, one or more parameters may be
   supplied.

      digest-algorithm = token

   The BNF for "parameter" is as is used in RFC 2616 [4].  All digest-
   algorithm values are case-insensitive.
```

but:

- provided no example of parameterized algorithms

In this draft we *preserved retained that sentence* but:

- we did not provide an example of parameterized algorithm;
- we did not update the reference to BNF to the latest http spec.

So, about your questions:

- it's not really about structured headers;
- iiuc RFC3230 already allowed parameters to the Digest headers, though
  no examples were provided.
